pom.xml

```diff
@@ -537,8 +537,8 @@ limitations under the License.
             <configuration>
               <rules>
                 <requireMavenVersion>
-                  <version>3.0.5</version>
-                  <message>This project requires at least Maven 3.0.5</message>
+                  <version>3.2.5</version>
+                  <message>This project requires at least Maven 3.2.5</message>
```

Comment:
It is requirement for build projects, so I would like see something newer, eg 3.6.x, 3.8.x

Reply:
In theory yes, but that will force us to drop 3.2.5 from ci test matrix

Reply:
We should agree on build matrix - compared to maintained Maven versions ... and so on ... but it is another discussion